util/util.go

```diff
@@ -60,18 +60,20 @@ func BuildClientConfig(client model.Client, server model.Server, setting model.G
 		peerPersistentKeepalive = fmt.Sprintf("PersistentKeepalive = %d\n", setting.PersistentKeepalive)
 	}
 
-	forwardMark := ""
-	if setting.ForwardMark != "" {
-		forwardMark = fmt.Sprintf("FwMark = %s\n", setting.ForwardMark)
+	/* // Don't add fwmark to client configs - it won't do anything useful
+	firewallMark := ""
+	if setting.FirewallMark != "" {
+		firewallMark = fmt.Sprintf("FwMark = %s\n", setting.FirewallMark)
 	}
+	*/
 
 	// build the config as string
 	strConfig := "[Interface]\n" +
 		clientAddress +
 		clientPrivateKey +
 		clientDNS +
 		clientMTU +
-		forwardMark +
+		//firewallMark +
```

Comment:
Where is the `firewallMark` used in the end?

Reply:
@systemcrash can you address my question? I don't know if I get the idea of having firewallMark since it is not used anywhere in the config file.